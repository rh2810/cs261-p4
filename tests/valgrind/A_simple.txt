==18464== Memcheck, a memory error detector
==18464== Copyright (C) 2002-2015, and GNU GPL'd, by Julian Seward et al.
==18464== Using Valgrind-3.11.0 and LibVEX; rerun with -h for copyright info
==18464== Command: ../y86 -e inputs/simple.o
==18464== 
==18464== 
==18464== HEAP SUMMARY:
==18464==     in use at exit: 0 bytes in 0 blocks
==18464==   total heap usage: 0 allocs, 0 frees, 0 bytes allocated
==18464== 
==18464== All heap blocks were freed -- no leaks are possible
==18464== 
==18464== For counts of detected and suppressed errors, rerun with: -v
==18464== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
