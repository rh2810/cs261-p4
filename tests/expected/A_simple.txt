Beginning execution at 0x0100
Y86 CPU state:
  %rip: 0000000000000000   flags: Z0 S0 O0     HLT
  %rax: 0000000000000005    %rcx: 0000000000000003
  %rdx: 0000000000000000    %rbx: 0000000000000000
  %rsp: 0000000000000f00    %rbp: 0000000000000000
  %rsi: 0000000000000000    %rdi: 0000000000000000
   %r8: 0000000000000000     %r9: 0000000000000000
  %r10: 0000000000000000    %r11: 0000000000000000
  %r12: 0000000000000000    %r13: 0000000000000000
  %r14: 0000000000000000
Total execution count: 7
